.travis.yml

```diff
@@ -13,8 +11,7 @@ cache:
     - ".eslintcache"
     - "node_modules"
 
-script: 
+script:
   - yarn run test-ci
   - (cd danger && yarn install)
   - yarn run danger
```

Comment:
I guess we don't need to run danger any longer and it's enough to do `run test-ci-partial` on travis.